localstack/services/apigateway/integration.py

```diff
@@ -196,27 +330,31 @@ class ResponseTemplates(Templates):
     Handles response template rendering
     """
 
-    def render(self, api_context: ApiInvocationContext):
-        response = api_context.response
+    def render(self, api_context: ApiInvocationContext, **kwargs):
```

Comment:
nit: could add the response type hint here (probably `bytes` or `Union[bytes, str]`?)